ui/deploy/default.conf

```diff
@@ -44,7 +44,7 @@ server {
 
     location /api/dojo {
         rewrite /api/dojo/(.*) /$1  break;
-        proxy_pass   http://${DOJO_HOST}:8000;
+        proxy_pass   http://${DATA_ANNOTATION_HOST}:8000;
```

Comment:
This, as written, will break when deployed in nginx because this will result in the full url being inside the url. Should probably just be:
```
proxy_pass   ${DATA_ANNOTATION_URL};
```

Reply:
Will do, that was a find - replace operation.